solaris/solaris10/generate_wazuh_packages.sh

```diff
@@ -163,7 +163,8 @@ package(){
     cd ${CURRENT_PATH}
     find /var/ossec | awk 'length > 0' > "wazuh-agent_$VERSION.list"
     ver=`echo $VERSION | cut -d'v' -f 2`
-    sed "s:VERSION=\".*\":VERSION=\"$ver\":g" pkginfo > pkginfo.new && mv pkginfo.new pkginfo
+    sed  "s:ARCH=\".*\":ARCH=\"$ARCH\":g" pkginfo > pkginfo.new && mv pkginfo.new pkginfo
```

Comment:
This sed looks good but, can you update this line https://github.com/wazuh/wazuh-packages/blob/7e78d1e481fde9fc76a0bf5e7b8b71a11dc5cddc/solaris/solaris10/generate_wazuh_packages.sh#L14? Is easier to get the architecture of a host running `uname -p`.

In addition, while revie

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/225/commits/8014515a1cb5bf5b1686ec51c7531518655dd816